tests/helm/remote_write_proxy/static/basic.output.yaml

```diff
@@ -0,0 +1,68 @@
+---
+# Source: sumologic/templates/metrics/remote-write-proxy/deployment.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: RELEASE-NAME-sumologic-remote-write-proxy
+  labels:
+    app: RELEASE-NAME-sumologic-remote-write-proxy
+    chart: "sumologic-2.4.0"
```

Comment:
Is that value correct? Don't we use any sort of placeholder for chart values in other tests?

Reply:
Good catch, switched to the placeholder.